sdk/parents/azure-arm-parent/pom.xml

```diff
@@ -57,7 +57,7 @@
     <reactor-netty.version>0.9.0.RELEASE</reactor-netty.version>
     <netty.version>4.1.42.Final</netty.version>
     <okhttp.version>4.2.2</okhttp.version>
-    <async-http-client.version>2.10.4</async-http-client.version>
+    <async-http-client.version>2.12.1</async-http-client.version>
```

Comment:
iirc. we don't update this one because it isn't managed by our versioning system yet. @JimSuplizio?

Reply:
I updated all of the async-http-clients with the older versions I could find, ultimately I only needed servicebus fixed as it was flagged by our IT Security Department for not being in compliance, and I didnt want to leave different versions of the same library in the repo.